Cargo.toml

```diff
@@ -59,14 +65,6 @@ indoc = "2.0.3"
 # dev dependencies
 mockito = { version = "1.6", default-features = false }
 
-[patch.crates-io]
-stacks-codec = { path = "./components/stacks-codec" }
-hiro-system-kit = { path = "./components/hiro-system-kit" }
-chainhook-sdk = { git = "https://github.com/hirosystems/chainhook.git", branch = "chore/update-clarinet-and-clarity" }
-chainhook-types = { git = "https://github.com/hirosystems/chainhook.git", branch = "chore/update-clarinet-and-clarity" }
-# chainhook-sdk = { path = "../chainhook/components/chainhook-sdk" }
-# chainhook-types = { path = "../chainhook/components/chainhook-types-rs" }
```

Comment:
Because everything is now only used internally, it doesn't need to be patched anymore